packages/utils/src/lib/reports/__snapshots__/log-stdout-summary.integration.test.ts.snap

```diff
@@ -36,22 +35,22 @@ ESLint audits
 ● Disallow reassigning \`const\` variables                              passed
 ● Disallow the use of \`debugger\`                                      passed
 ● Disallow the use of undeclared variables unless mentioned in        passed
-  \`/*global */\` comments
+\`/*global */\` comments
 ● Disallow undeclared variables in JSX                                passed
 ● Disallow unescaped HTML entities from appearing in markup           passed
 ● Disallow usage of deprecated methods                                passed
 ● Disallow usage of findDOMNode                                       passed
 ● Disallow usage of isMounted                                         passed
 ● Disallow usage of the return value of ReactDOM.render               passed
 ● Disallow usage of unknown DOM property                              passed
-● Disallow use of optional chaining in contexts where the             passed
-  \`undefined\` value is not allowed
+● Disallow use of optional chaining in contexts where the \`undefined\` passed
+value is not allowed
 ● Disallow using Object.assign with an object literal as the first    passed
-  argument and prefer the use of object spread instead
+argument and prefer the use of object spread instead
 ● Disallow using string references                                    passed
 ● Disallow variables used in JSX to be incorrectly marked as unused   passed
 ● Disallow when a DOM element is using both children and              passed
-  dangerouslySetInnerHTML
+dangerouslySetInnerHTML
```

Comment:
:lady_beetle: Indentation is longer preserved in wrapped rows :disappointed:

Reply:
THX for spotting should be fixed now